core/server/master/src/test/java/alluxio/master/journal/sink/JournalSinkTest.java

```diff
@@ -240,6 +314,29 @@ private long findRename(String newName) throws Exception {
     return INVALID_ID;
   }
 
+  private long findRename(String newName, String path, String newPath) throws Exception {
+    while (!mEntries.isEmpty()) {
+      JournalEntry entry = mEntries.poll();
+      if (entry.hasRename() && entry.getRename().getNewName().equals(newName)
+          && entry.getRename().getNewPath().equals(newPath)
+          && entry.getRename().getPath().equals(path)) {
+        return entry.getRename().getId();
+      }
+    }
+    return INVALID_ID;
+  }
+
+  private long findCompleteFile(long id, String path) throws Exception {
+    while (!mEntries.isEmpty()) {
+      JournalEntry entry = mEntries.poll();
+      if (entry.hasUpdateInodeFile() && entry.getUpdateInodeFile().getId() == id
```

Comment:
shouldn't this check for the `isCompleted` flag to be set and `true`?

Reply:
Done.